resources-service/tests/service/resources.test.ts

```diff
@@ -0,0 +1,49 @@
+import { mockingProxy, mockSuccessfulTwilioAuthentication } from '@tech-matters/testing';
+import { headers, getRequest, getServer } from './server';
+import { db } from '../../src/connection-pool';
+
+export const workerSid = 'WK-worker-sid';
+
+const server = getServer();
+const request = getRequest(server);
+
+afterAll(done => {
+  mockingProxy.stop().finally(() => {
+    server.close(done);
+  });
+});
+
+beforeAll(async () => {
+  await mockingProxy.start();
+  await mockSuccessfulTwilioAuthentication(workerSid);
+  await db.multi(
+    [
+      'INSERT INTO resources."Resources" (id, "accountSid", "name") VALUES (\'RESOURCE_1\', \'ACCOUNT_1\', \'Resource 1 (Account 1)\')',
```

Comment:
Totes personal preference, but would back-ticks allow us to avoid having to escape quotes without breaking lint?